opencensus/plugins/grpc/grpc_plugin.h

```diff
@@ -24,6 +29,9 @@ namespace opencensus {
 // defined below.
 void RegisterGrpcPlugin();
 
+// Returns the tracing Span for the current RPC.
+opencensus::trace::Span GetSpanFromServerContext(grpc::ServerContext* context);
```

Comment:
This comes from https://github.com/census-instrumentation/opencensus-cpp/pull/60